app/helpers/pretty-bytes.js

```diff
@@ -2,4 +2,17 @@ import { helper } from '@ember/component/helper';
 
 import prettyBytes from 'pretty-bytes';
 
-export default helper(([bytes], options) => prettyBytes(bytes, options));
+/**
+ * See https://github.com/rust-lang/crates.io/discussions/7177
+ *
+ * Here we set fraction digits to 1 because `cargo publish`
+ * uses this format (see https://github.com/rust-lang/cargo/blob/master/src/cargo/ops/cargo_package.rs#L168-L171)
```

Comment:
Could you please use a tag instead of the master branch here?
Using a fixed version can make sure we can always find it.

Reply:
I see! I'll fix them...